### PR TITLE
Added bamboo as another ci service

### LINKED ISSUE #4: bamboo support
https://www.atlassian.com/software/bamboo
🌷

### PR DESCRIPTION
**Pull request by maraisr**
Resolves #4

### CODE REVIEW

File: lib/bamboo.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// https://confluence.atlassian.com/bamboo/bamboo-variables-289277087.html
+
+module.exports = {
+  detect() {
+    return Boolean(process.env.bamboo_agentId);
+  },
+  configuration() {
+    return {
+      service: 'bamboo',
+      commit: process.env.bamboo_planRepository_1_revision,
```
Comment (maraisr) [inline lib/bamboo.js]: I know this isn't camelCase, but Bamboo doesn't give these env vars as uppercase.... Can we ignore this with `xo`?

File: README.md [hunk 2]
```diff
@@ -52,5 +52,6 @@ const {isCi, service, commit, build, branch, job, pr, isPr, slug, root} = envCi(
 | [Shippable](http://docs.shippable.com/ci/env-vars/#stdEnv)                                                | `shippable` |   ✅    |    ✅     |    ✅    |    ✅     |   ✅   |  ✅   |   ✅    |   ✅    |   ✅    |
 | [Travis](https://docs.travis-ci.com/user/environment-variables)                                           |  `travis`   |   ✅    |    ✅     |    ✅    |    ✅     |   ✅   |  ✅   |   ✅    |   ✅    |   ✅    |
 | [Wercker](http://devcenter.wercker.com/docs/environment-variables/available-env-vars#hs_cos_wrapper_name) |  `wercker`  |   ✅    |    ✅     |    ✅    |    ✅     |   ❌   |  ❌   |   ❌    |   ✅    |   ✅    |
+| [Bamboo](https://confluence.atlassian.com/bamboo/bamboo-variables-289277087.html)                         |  `bamboo`   |   ✅    |    ✅     |    ✅    |    ✅     |   ✅   |  ❌   |   ❌    |   ❌    |   ✅    |
```
Comment (azz) [inline README.md]: These are sorted alphabetically.

Comment (maraisr) [reply README.md]: Resolved.

File: lib/bamboo.js (hunk 1 above)
Comment (pvdlg) [reply lib/bamboo.js]: `xo` works with ESLint. You can use the same type of comment to disable a rule: https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments

**Comment by pvdlg** [comment]
Could you also add `bamboo` to the `package.json` keywords, ordered alphabetically? Thanks

File: package.json [hunk 3]
```diff
@@ -31,6 +31,7 @@
   "keywords": [
     "appveyor",
     "buildkite",
+    "bamboo",
```
Comment (pvdlg) [inline package.json]: Sorry for nitpicking. `bamboo` should come before `buildkite`.
I try to push the fix myself but I don't have permission on your branch.
If you give me permission with I can do it myself: https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

Comment (maraisr) [reply package.json]: @pvdlg all good! Fixed up! 🎉

**Comment by pvdlg** [comment]
Thank you!

**Pull request merged**